tests/Doctrine/Tests/DBAL/Functional/WriteTest.php

```diff
@@ -276,11 +308,15 @@ public function testEmptyIdentityInsert() : void
 
         $this->connection->exec($sql);
 
-        $firstId = $this->lastInsertId($seqName);
+        $firstId = $seqName !== null
```

Comment:
No conditionals in tests please.

Reply:
Fixed in 334cf158f5875701df77f8778faaa40fe73ba2c4.